core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/store/dot-palette.store.ts

```diff
@@ -220,8 +220,14 @@ export class DotPaletteStore extends ComponentStore<DotPaletteState> {
                 ])
                     .pipe(take(1))
                     .subscribe((results: DotCMSContentType[][] ) => {
+                        const [ allowContent, widgets ] = results;
+
+                        // Some pages allow widgets, but we already make a request just to get them.
```

Comment:
```suggestion
                        // All pages allow widgets, but we already make a request just to get them.
```

Reply:
Could it be something like:

> Some pages bring widgets in the `CONTENT_PALETTE_HIDDEN_CONTENT_TYPES`, and others don't. However, all pages allow widgets, so we make a request just to get them.

That comment is because here, some pages hide the widgets and others don't.

https://github.com/dotCMS/core/blob/64bbc45343f85bc5df6122e87b25195ea1b2d7f1/core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts#L261-L269

Reply:
Could it be something like:

> Some pages bring widgets in the `CONTENT_PALETTE_HIDDEN_CONTENT_TYPES`, and others don't. However, all pages allow widgets, so we make a request just to get them.

That comment is because here, some pages hide the widgets and others don't.

https://github.com/dotCMS/core/blob/64bbc45343f85bc5df6122e87b25195ea1b2d7f1/core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts#L261-L269

Reply:
Perfect!